Install not working for me

Hey there

I am completely new to Docpad so this could be purely user error on my part. When I try to install this plugin using the following command:

```
docpad install swig
```

I get the following error:

```
npm http 304 https://registry.npmjs.org/docpad-plugin-swig
npm ERR! Error: No compatible version found: docpad-plugin-swig@'>=2.0.0-0 <3.0.0-0'
npm ERR! Valid install targets:
npm ERR! ["0.0.1","0.0.2","0.0.3"]
npm ERR!     at installTargetsError (/usr/share/npm/lib/cache.js:719:10)
npm ERR!     at next (/usr/share/npm/lib/cache.js:698:17)
npm ERR!     at /usr/share/npm/lib/cache.js:675:5
npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:142:7)
npm ERR!     at /usr/lib/nodejs/graceful-fs/polyfills.js:133:7
npm ERR!     at Object.oncomplete (fs.js:107:15)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Linux 3.13.0-32-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "docpad-plugin-swig@2" "--save"
npm ERR! cwd /var/www/docpad.dev
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.10
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /var/www/docpad.dev/npm-debug.log
npm ERR! not ok code 0
error: Something went wrong with the action
error: An error occured: 
Error: npm http GET https://registry.npmjs.org/docpad-plugin-swig
npm http 304 https://registry.npmjs.org/docpad-plugin-swig
npm ERR! Error: No compatible version found: docpad-plugin-swig@'>=2.0.0-0 <3.0.0-0'
npm ERR! Valid install targets:
npm ERR! ["0.0.1","0.0.2","0.0.3"]
npm ERR!     at installTargetsError (/usr/share/npm/lib/cache.js:719:10)
npm ERR!     at next (/usr/share/npm/lib/cache.js:698:17)
npm ERR!     at /usr/share/npm/lib/cache.js:675:5
npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:142:7)
npm ERR!     at /usr/lib/nodejs/graceful-fs/polyfills.js:133:7
npm ERR!     at Object.oncomplete (fs.js:107:15)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Linux 3.13.0-32-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "docpad-plugin-swig@2" "--save"
npm ERR! cwd /var/www/docpad.dev
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.10
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /var/www/docpad.dev/npm-debug.log
npm ERR! not ok code 0

    at ChildProcess.<anonymous> (/var/www/docpad.dev/node_modules/safeps/out/lib/safeps.js:165:23)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:743:16)
    at Process.ChildProcess._handle.onexit (child_process.js:810:5)
info: Shutting down... cya next time!
```

Any ideas?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "docpad-plugin-swig",
-	"version": "0.0.3",
+	"version": "2.0.3",
 	"description": "Adds support for the swig templating engine to DocPad.",
 	"homepage": "http://github.com/thisispete/docpad-plugin-swig",
 	"keywords": [

